README.md

```diff
@@ -1,113 +1,32 @@
-<p align="center">
-  <a href="https://www.gatsbyjs.com">
-    <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
-  </a>
-</p>
-<h1 align="center">
-  Starter for a Gatsby Plugin
-</h1>
+# gatsby-source-apostrophe
 
-A minimal boilerplate for the essential files Gatsby looks for in a plugin.
+Source plugin for accessing ApostropheCMS content APIs in Gatsby
 
-## 🚀 Quick start
+## Install
 
-To get started creating a new plugin, you can follow these steps:
+`npm install gatsby-source-apostrophe`
 
-1. Initialize a new plugin from the starter with `gatsby new`
-
-```shell
-gatsby new my-plugin https://github.com/gatsbyjs/gatsby-starter-plugin
-```
-
-If you already have a Gatsby site, you can use it. Otherwise, you can [create a new Gatsby site](https://www.gatsbyjs.com/tutorial/part-zero/#create-a-gatsby-site) to test your plugin.
-
-Your directory structure will look similar to this:
-
-```text
-/my-gatsby-site
-├── gatsby-config.js
-└── /src
-    └── /pages
-        └── /index.js
-/my-plugin
-├── gatsby-browser.js
-├── gatsby-node.js
-├── gatsby-ssr.js
-├── index.js
-├── package.json
-└── README.md
-```
-
-With `my-gatsby-site` being your Gatsby site, and `my-plugin` being your plugin. You could also include the plugin in your [site's `plugins` folder](https://www.gatsbyjs.com/docs/loading-plugins-from-your-local-plugins-folder/).
-
-2. Include the plugin in a Gatsby site
-
-Inside of the `gatsby-config.js` file of your site (in this case, `my-gatsby-site`), include the plugin in the `plugins` array:
+## How to use
 
 ```javascript
+// In your gatsby-config.js
 module.exports = {
   plugins: [
-    // other gatsby plugins
-    // ...
-    require.resolve(`../my-plugin`),
-  ],
+    {
+      resolve: `gatsby-source-apostrophe`,
+      options: {
+        apiKey: 'my-apos-api-key',
+        baseUrl: 'http://localhost:3000',
+        pieceTypes: [
+          '@apostrophecms/user',
```

Comment:
People might just copy and paste this and then they'd have an API with a lot of security implications activated. Maybe mention that it is possible later instead.

Reply:
A pointer to the right page of the A3 docs for configuring api keys for piece types might be helpful.